packages/BetterReflection/src/SourceLocator/SourceLocatorFactory.php

```diff
@@ -66,7 +66,7 @@ public function createWithSource(array $source): SourceLocator
      */
     private function createCommonLocators(): array
     {
-        if ($this->commonLocators) {
+        if (count($this->commonLocators) > 0) {
```

Comment:
This rule seems good, but only for adding `count()`

The `> 0` part is not context aware, thus not needed